README.md

```diff
@@ -148,70 +171,22 @@ be defined with the environment variable `NJS_ACME_ACCOUNT_EMAIL`.
   ssl_certificate data:$dynamic_ssl_cert;
   ssl_certificate_key data:$dynamic_ssl_key;
   ```
-* You may use `js_shared_dict_zone` to improve performance and cache certificates
-and keys in shared memory, then set variable with the zone name.
-  ```nginx
-  js_var $njs_acme_shared_dict_zone_name acme;
-  ```
 
 ### `location` Blocks
-* Location to handle ACME challenge requests.
+* Location to handle ACME challenge requests. This must be accessible from the ACME server - in most cases this means accessbile from another host on the Internet if you are using a service like Let's Encrypt.
   ```nginx
   location ~ "^/\.well-known/acme-challenge/[-_A-Za-z0-9]{22,128}$" {
     js_content acme.challengeResponse;
   }
   ```
-
-* Location, that when requested, inspects the stored certificate (if present) and will request a new certificate if necessary. The included `docker-compose.yml` shows how to use a `healthcheck:` configuration for the NGINX service to periodically request this endpoint.  This should usually be restricted to local or Docker network requests to avoid it being abused. NGINX's [ngx_http_access_module](http://nginx.org/en/docs/http/ngx_http_access_module.html) provides functionality to allow or deny requests. You can look for requests to `/acme/auto` in the NGINX access log entries to find the client IP address to allow.
+* Named location to contain the `js_periodic` directive to automatically
+request or renew certificates if necessary.
   ```nginx
-  location = /acme/auto {
-    js_content acme.clientAutoMode;
-    # Make sure you are not exposing this location to the Internet.
-    # Allow only the IP or IP range of the host making automated /acme/auto requests.
-    # You will probably need to customize the `allow` address(es) below.
-    # allow 192.168.1.0/24;
-    # allow 2001:0db8::/32;
-    # deny all;
+  location @acmePeriodicAuto {
```

Comment:
Is this location ever referenced? Or are we just doing this because `js_periodic` requires that it be located in a location block and we want it not to be confused with something that can be accessed?   I can't seem to find any reference to this named location anywhere in `main` or in this PR which makes think it's just to provide a `location`.  If that's the case I think we should explain here as well as in the example that the location itself is not significant.

Reply:
Thanks! Updated with more context.